Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mitmproxy/mitmproxy:latest
+
+COPY ./ /src/tuf-mitm-proxy
+
+RUN apk add --no-cache \
```

Comment:
Check out `apk`'s `--virtual` flag, it makes cleaning up (`apk del`) a bit easier.